ecc-functions/owners-transfercontext/__init__.py

```
@@ -0,0 +1,22 @@
+    with urllib.request.urlopen(req, timeout=15) as r:
+        return json.loads(r.read().decode("utf-8") or "null")
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    try:
+        src = req.params.get("sourceOwnerId")
+        if not src:
+            return func.HttpResponse(json.dumps({"error": "missing_sourceOwnerId"}), status_code=400, mimetype="application/json")
+        owner = _sb("owners", {"id": f"eq.{src}", "select": "id,display_name,primary_email,primary_phone"})
+        ctx = {"sourceOwner": (owner[0] if isinstance(owner, list) and owner else owner), "canTransfer": bool(owner)}
+        return func.HttpResponse(json.dumps(ctx), mimetype="application/json")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)